spec/features/movies/show_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+RSpec.describe "Movies", type: :feature do
+  describe "Movies Show" do
+    it "displays the movies title, creation year, and genre" do
+    studio1 = Studio.create!(name: "Universal Studios", location: "Hollywood")
+    movie1 = Movie.create!(title: "Toy Story", creation_year: "1995", genre: "Animation", studio: studio1)
+    movie2 = Movie.create!(title: "The Godfather", creation_year: "1972", genre: "Crime", studio: studio1)
+    actor1 = Actor.create!(name: "Tom Hanks", age: 65)
+    actor2 = Actor.create!(name: "Tim Allen", age: 70)
+    actor3 = Actor.create!(name: "Al Capone", age: 90)
+    # movie1.actors << [actor1, actor2, actor3]
```

Comment:
I think this line should work once you have the other changes in place that I recommended, and then this test should cover what this user story needs.